nextcord/guild.py

```diff
@@ -1349,6 +1354,27 @@ async def create_stage_channel(
         position: :class:`int`
             The position in the channel list. This is a number that starts
             at 0. e.g. the top channel is position 0.
+        bitrate: Optional[:class:`int`]
+            The channel's preferred audio bitrate in bits per second.
+
+            .. versionadded:: 2.5
+        user_limit: :class:`int`
+            The channel's limit for number of members that can be in a voice channel.
+
+            .. versionadded:: 2.5
```

Comment:
Does a user upper limit need/should to be documented here? IIRC there is one but I am possibly wrong in this regard